eng/targets/XUnit.targets

```diff
@@ -6,6 +6,15 @@
     <PrepareForBuildDependsOn>$(PrepareForBuildDependsOn);AddDefaultTestAppConfig</PrepareForBuildDependsOn>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(TargetFrameworkIdentifier)' != '.NETFramework'">
+    <!--
+      In newer versions of the .NET SDK a deps.json file is used instead of copying 
+      dependencies to the output directory. Because we copy and run tests across different
+      machines, we need to ensure that the dependencies are copied to the output directory.
+    -->
+    <CopyLocalLockFileAssemblies>true</CopyLocalLockFileAssemblies>
```

Comment:
Does this property need to be conditional?

Reply:
it doesn't apply to .NET Framework, so I just reflexively did not include it. I can test what the build output is with this universally applied to test projects.